sources/unstructured_data/inbox/settings.py

```diff
@@ -0,0 +1,2 @@
+STORAGE_FOLDER_PATH = "temp"
```

Comment:
use `tempfile` to get temp folder

Reply:
i can use it in unstructured_data_pipeline.py script

Reply:
I use it in tests